### PR TITLE
Add visually hidden text to LinkContents for PGL.

### LINKED ISSUE #2849: Add photogallery offscreen text
**Is your feature request related to a problem? Please describe.**
We are now displaying PGL promos in our Front Pages but the offscreen text is not added in the headline. 

The problem is that in `LinkContents` we are just looking for the `MAP` CPS type.

**Describe the solution you'd like**
Check if the item passed to the `LinkContents` component is a `PGL` and add the same logic as in the `MediaIndicator` to display the offscreen text also when it is a `photogallery`.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #2849 

**Overall change:** _Adds photo gallery offscreen text._

**Code changes:**

- _Add code to also show visually hidden text ._
- _Add unit test._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/StoryPromo/LinkContents/index.test.jsx [hunk 1]
```diff
@@ -34,6 +34,16 @@ const mediaItem = {
   },
 };
 
+const photogalleryItem = {
```
Comment (DenisHdz) [inline src/app/containers/StoryPromo/LinkContents/index.test.jsx:55]: For some reason this file doesn't contain what has been merged when adding the PGL to the whitelist: 
https://github.com/bbc/simorgh/blob/ff80eea8842dae7ec864538443c330600cbc589f/src/app/containers/StoryPromo/MediaIndicator/index.test.jsx

**Review by DenisHdz — CHANGES_REQUESTED** [review]

File: src/app/containers/StoryPromo/LinkContents/index.test.jsx (hunk 1 above)
Comment (Bopchy) [reply src/app/containers/StoryPromo/LinkContents/index.test.jsx:55]: I'm a bit confused about where the difference is. I have updated the branch though - is the issue still there?

Comment (DenisHdz) [reply src/app/containers/StoryPromo/LinkContents/index.test.jsx:55]: My bad. Confused this file with `MediaIndicator/index.test.jsx`

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Review by j-pendlebury — APPROVED** [review]
Looks good 👍

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**